src/Spice86.Core/Emulator/Memory/Memory.cs

```diff
@@ -175,5 +179,8 @@ public class Memory {
     private void MonitorWriteAccess(uint address, byte value) {
         CurrentlyWritingByte = value;
         _writeBreakPoints.TriggerMatchingBreakPoints(address);
+        if (_machine != null && address is >= 0xA0000 and <= 0xBFFFF) {
```

Comment:
Replace with private constants for 0xA0000 and 0xBFFFF.

Aren't those values dependant upon the current video mode ? Is it the same range for EGA, CGA, Text modes... ?

Reply:
This is a hack that works for now. It is meant to be replaced once we have something better.
I should have commented this.

Reply:
Comment added